tools/traffic/config/flags.go

```diff
@@ -0,0 +1,340 @@
+package config
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/urfave/cli"
+)
+
+const (
+	FlagPrefix = "traffic-generator"
+	envPrefix  = "TRAFFIC_GENERATOR"
+)
+
+var (
```

Comment:
Instead of listing flags for dependent packages again, we could import the flags directly like [here](https://github.com/Layr-Labs/eigenda/blob/master/retriever/flags/flags.go#L99-L103)

Reply:
change made